validator/sawtooth_validator/execution/predecessor_tree.py

```diff
@@ -37,7 +37,7 @@ def __repr__(self):
         return repr(retval)
 
 
-class PredecessorTree:
+class Tree:
```

Comment:
What we want to end up with here is a generic RadixTrie implementation, so 'RadixTree' or 'RadixTrie' would be sensible names.

Reply:
The name can be changed when the data structure changes. Alternatively, the name can be left vague, and the specifics of the data structure can be changed without the possibility of rendering the name inaccurate.